Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -158,27 +158,50 @@ fi
 
 # ArduPilot official Toolchain for STM32 boards
 function install_arm_none_eabi_toolchain() {
-  # GNU Tools for ARM Embedded Processors
-  # (see https://launchpad.net/gcc-arm-embedded/)
-  ARM_ROOT="gcc-arm-none-eabi-10-2020-q4-major"
-  ARM_TARBALL="$ARM_ROOT-x86_64-linux.tar.bz2"
-  ARM_TARBALL_URL="https://firmware.ardupilot.org/Tools/STM32-tools/$ARM_TARBALL"
-  if [ ! -d $OPT/$ARM_ROOT ]; then
-    (
-        cd $OPT;
-        heading "Installing toolchain for STM32 Boards"
-        echo "Downloading from ArduPilot server"
-        sudo wget $ARM_TARBALL_URL
-        echo "Installing..."
-        sudo tar xjf ${ARM_TARBALL}
-        echo "... Cleaning"
-        sudo rm ${ARM_TARBALL};
-    )
-  fi
-  echo "Registering STM32 Toolchain for ccache"
-  sudo ln -s -f $CCACHE_PATH /usr/lib/ccache/arm-none-eabi-g++
-  sudo ln -s -f $CCACHE_PATH /usr/lib/ccache/arm-none-eabi-gcc
-  echo "Done!"
+    # GNU Tools for ARM Embedded Processors
+    # (see https://launchpad.net/gcc-arm-embedded/)
+    ARM_ROOT="gcc-arm-none-eabi-10-2020-q4-major"
+    case $(uname -m) in
+        x86_64)
+            if [ ! -d $OPT/$ARM_ROOT ]; then
+                (
+                    cd $OPT
+                    heading "Installing toolchain for STM32 Boards"
+                    echo "Installing toolchain for STM32 Boards"
+                    echo "Downloading from ArduPilot server"
+                    sudo wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-x86_64-linux.tar.bz2
```

Comment:
this should use ardupilot server

Reply:
Both wgets in here seem to work.  Not sure if someone dropped the tarball in there after seeing this...